### PR TITLE
update DALEX with mlr

### PR DESCRIPTION
**Pull request by kozaka93**
update vignette #43

### CODE REVIEW

File: DALEX_mlr.Rmd [hunk 1]
```diff
@@ -0,0 +1,342 @@
...
+To illustrate applications of *DALEX* to classification problems we will use a wine dataset available in the *breakDown* package. We want to classify the quality of wine. Originally this variable has 7 levels but in our example, it will be reduced to the binary classification. Our classification will be based on eleven features from this data set.
+
+White wine quality data is related to variants of the Portuguese "Vinho Verde" wine. For more details, consult: http://www.vinhoverde.pt/en/.
+
+```{r}
+data(wine)
+wine$quality <- ifelse(wine$quality>5, 1, 0)
+```
+
+First, we create a train and test indexes which ones are needed to train the *mlr* models when we don't have an additional test set - `wineTest`.
```
Comment (jakwisn) [inline DALEX_mlr.Rmd:224]: ... which ~~ones~~ are needed...

File: DALEX_mlr.Rmd [hunk 2]
```diff
@@ -0,0 +1,342 @@
...
+
+plot(vi_classif_rf, vi_classif_glm, vi_classif_svm)
+```
+
+Left edges of intervals start in full model. Length of the interval coresponds to a variable importance. Longer interval means larger loss, so the variable is more important.
+
+
+## Model profile
+
+As previously we create explainers which are designed to better understand the relation between a variable and model output: PDP plots and ALE plots.
```
Comment (jakwisn) [inline DALEX_mlr.Rmd:312]: As previously **,** - with comma

Comment (kozaka93) [reply DALEX_mlr.Rmd:312]: Ok, I fix this.

File: DALEX_mlr.Rmd (hunk 1 above)
Comment (kozaka93) [reply DALEX_mlr.Rmd:224]: Ok, I fix this.

File: DALEX_mlr.Rmd [hunk 3]
```diff
@@ -0,0 +1,342 @@
+---
+title: "How to use DALEX with mlr"
+author: 
+  - Aleksandra Grudziąż, Anna Kozak
+date: First created on April 28, 2018. Updated on Nov 07, 2020.
+output:
```
[the remark this answers is not in the document]
Comment (kozaka93) [reply DALEX_mlr.Rmd:6]: Yes. I closed this pr. Thanks

**Pull request closed without merging**